ql/pom.xml

```diff
@@ -133,6 +133,11 @@
       <artifactId>log4j-1.2-api</artifactId>
       <version>${log4j2.version}</version>
     </dependency>
+    <dependency>
```

Comment:
log4j-slf4j-impl has a transitive dependency for log4j-core, may be we do not need this?

Reply:
fixed.